core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -77,91 +68,60 @@ export class SplashPageComponent implements OnInit {
     );
   }
 
-  private _nagivateToClassroomPage(): void {
+  private _navigateToClassroomPage(): void {
     this.windowRef.nativeWindow.location.href = this.classroomUrl;
   }
 
+  private _navigateToAccountCreationPage(): void {
+    this.userService.getLoginUrlAsync().then(loginUrl => {
+      if (loginUrl) {
+        setTimeout(() => {
```

Comment:
Could you please explain why adding timeout here?

Reply:
This timeout thing is there at few places in the codebase where we navigate to Google 'sign-in' page.

For Eg [here](https://github.com/search?q=repo%3Aoppia%2Foppia%20this.userService.getLoginUrlAsync().then(loginUrl%20%3D%3E%20%7B&type=code)

I guess, this 0.15 second delay is for waiting for Google 'Sign in' to load fully